tidy3d/components/spice/sources/ac.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+from typing import Optional
+
+import pydantic.v1 as pd
+
+from tidy3d.components.base import Tidy3dBaseModel
+from tidy3d.components.types import ArrayFloat1D
+from tidy3d.constants import HERTZ, VOLT
+from tidy3d.constants import inf as td_inf
+
+
+class ACVoltageSource(Tidy3dBaseModel):
+    """
+    Small-signal AC voltage source.
+
+    Notes
+    -----
+    This source represents a small-signal AC excitation defined by a list of bias voltages,
+    a set of small-signal analysis frequencies, and a small-signal amplitude (linear scale).
+
+    The bias ``voltage`` refers to the DC operating point above the simulation ground. Currently, electrical ports are not defined.
+
+    Examples
+    --------
+    >>> import tidy3d as td
+    >>> voltages = [-0.5, 0, 1, 2, 3, 4]
+    >>> frequency = [1e3, 1e4, 1e5]
+    >>> amplitude = 1e-3
+    >>> voltage_source = td.ACVoltageSource(
+    ...     voltage=voltages,
+    ...     frequency=frequency,
+    ...     amplitude=amplitude,
+    ... )
+    """
+
+    name: Optional[str]
```

Comment:
What is the purpose of the `name`?

If it's needed, it should be formatted similarly to e.g. [this](https://github.com/flexcompute/tidy3d/blob/6e1ff2249daac150d12ffd043b52567ac2655db8/tidy3d/components/base_sim/monitor.py#L20-L25)